src/radical/pilot/task_description.py

```diff
@@ -26,6 +27,9 @@
 ARGS             = 'args'
 KWARGS           = 'kwargs'
 
+# mode: TASK_PY_FUNCTION
```

Comment:
It would make sense, but your suggestion was to make a totally different type of task and I ended up creating `PY_FUCNTION`, so I do not know how to proceed now.

Reply:
Ha, that puts you in a pickle, and me too! :-)  Let me look over the code again...